pkg/apiserver/info/info.go

```diff
@@ -28,8 +28,7 @@ import (
 	"github.com/pingcap/tidb-dashboard/pkg/config"
 	"github.com/pingcap/tidb-dashboard/pkg/dbstore"
 	"github.com/pingcap/tidb-dashboard/pkg/tidb"
-	"github.com/pingcap/tidb-dashboard/pkg/utils"
-	"github.com/pingcap/tidb-dashboard/pkg/utils/version"
+	"github.com/pingcap/tidb-dashboard/util/versionutil"
```

Comment:
Oh. I forgot to list when util should be called "xxx" or "xxxutil". My current convention is that:

If there is a commonly used package called "xxx", then we should use "xxxutil" to avoid naming conflict. For example, if we want to have our own tls util, since there is a [std tls package](https://pkg.go.dev/crypto/tls), we should call it "tlsutil".

Otherwise it would be just fine to call "xxx".

You can name it as "version" I think. There is no name conflict.